knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml

```diff
@@ -767,6 +834,66 @@ data:
       </root>
     </configuration>
 ---
+# Copyright 2019 The Knative Authors
+#
```

Comment:
Note: we should remove this from midstream 1.3

Reply:
_this_ ?

Reply:
yeah, we used to remove that via patch